src/common/Widget.js

```diff
@@ -122,7 +122,7 @@
     };
 
     // Serialization  ---
-    Widget.prototype.require = require || function (paths, cb) {
+    Widget.prototype.require = window.require || function (paths, cb) {
```

Comment:
What I was trying to say is that here:  https://github.com/hmsonline/Visualization/blob/GORDON_DEV_FIX/src/common/Widget.js#L6

You should change to: 
```
root.common_Widget = factory(root.d3, window.require);
```